cypress/integration/contact/contact.test.js

```diff
@@ -0,0 +1,60 @@
+/// <reference types="cypress" />
+import {
+  id,
+  telephoneNumber,
+  email,
+  textString,
+  phoneNumber,
+  uaSchedule,
+  enSchedule,
+  uaAddress,
+  enAddress
+} from './contact.variables';
+
+describe('Contacts test', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('ADMIN_LOGIN'), Cypress.env('ADMIN_PASSWORD'));
+    cy.visit('/contacts');
+    cy.wait(3000);
+    cy.get('.MuiTableCell-root.MuiTableCell-body').as('table');
+  });
+
+  it('Should find a page title', () => {
+    cy.contains('Інформація про контакти');
+  });
+
+  it('User list row should have all types of necessary information about the user', () => {
+    cy.get('@table').eq(0).invoke('text').should('match', telephoneNumber);
+    cy.get('@table').eq(1).invoke('text').should('match', email);
+    cy.get('@table').eq(2).invoke('text').should('match', textString);
+  });
+
+  it('Data from the server should be equal with the incoming data', () => {
+    cy.visit(`contacts/${id}`);
```

Comment:
Why are you using visit when you can mock responses and use  `cy.get(selector).click()`